lib/apartment/active_record/log_subscriber.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Style/StringLiterals, Style/BlockDelimiters
```

Comment:
If possible i'd like to avoid disabling of rubocop rules. I'm actually trying to cleanup the rubocop todo and have a standard set for the gem. so if we can avoid adding more disables would be good.

Reply:
I Agree, the rubocop is enable.